### PR TITLE
"packs" directory auto-create

### PR DESCRIPTION
**Pull request by Vesek**
"packs" directory was not created automaticaly and geyser crashed/disabled. Added fix for that.

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java [hunk 1]
```diff
@@ -32,7 +32,10 @@ public static void loadPacks() {
         }
 
         File directory = new File("packs");
-
+		
+		if(!directory.exists()){
+			directory.mkdir();
+		}
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java:38]: Please cleanup indentation

File: connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java [hunk 2]
```diff
@@ -32,7 +32,10 @@ public static void loadPacks() {
         }
 
         File directory = new File("packs");
-
+		
+		if(!directory.exists()){
```
Comment (rtm516) [inline connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java:36]: Please add spacing around brackets `if (!directory.exists()) {`

Comment (Vesek) [reply connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java]: Both changed!

File: connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java [hunk 3]
```diff
@@ -33,6 +33,10 @@ public static void loadPacks() {
 
         File directory = new File("packs");
 
+	if (!directory.exists()) {
+	    directory.mkdir();
+	}
+
```
Comment (RednedEpic) [inline connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java:39]: Indentation is still a bit off (use spaces instead of tabs).

Comment (Vesek) [reply connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java]: I hope it's right now.

**Pull request merged**